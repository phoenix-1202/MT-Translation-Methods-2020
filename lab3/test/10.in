randomFunction :: Integer -> Double -> Point -> Point
randomFunction a b c = ((( + (Point a b) (Point (leftProjection b) (rightProjection c))))) / (b)