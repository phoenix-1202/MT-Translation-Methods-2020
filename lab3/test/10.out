#include <bits/stdc++.h>;
using namespace std;

Point randomFunction(long long arg1, double arg2, Point arg3) {
return ((Point(arg1, arg2) + Point(leftProjection(arg2), rightProjection(arg3))) / arg2);
}

//EXIT